api/controller/utils/qrCodeUtil.js

```diff
@@ -0,0 +1,13 @@
+var qrcode = require('qrcode');
+
+exports.generateQRCode = async function (otpAuth) {
+
+	try {
+		var QRCodeImageUrl = await qrcode.toDataURL(otpAuth);
+		return {image: `<img src='${QRCodeImageUrl}' alt='qr-code-img' />`};
+	} catch (error) {
+		console.log('Could not generate QR code', error);
+		return;
```

Comment:
Empty return statement will crash the code.

Handle `else` condition properly.
Maybe you can use something like `<img alt='could-not-generate-qr' />` in else. Try, test, and handle else condition.